docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,489 @@
+# Porting PSA targets
+
+This document describes the process of adding new PSA targets to Mbed OS, focusing on target configurations, build and validation. Mbed OS relies on [TF-M](https://git.trustedfirmware.org/trusted-firmware-m.git/tree/) for PSA Services and SPM, so the PSA target you are adding to Mbed OS **must** already be supported by TF-M.
```

Comment:
We wouldn't expect readers to know what SPM is. Instead of "PSA Services and SPM", could we instead say "for its implementation of PSA"?

Reply:
This is because the glossary is after the first paragraph; Dev originally put it before the content, but I found it distracting.

Reply:
I don't think we need to explain SPM either. That's an unnecessary detail for porting targets and we don't need to mention SPM here or in any glossary.

Reply:
fixed